### DISCUSSION TITLE
Welcome to bot-whatsapp Discussions!

### DISCUSSION THREAD

**Discussion by dngda** [dngda/bot-whatsapp#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dstwn** [comment]
I want to add the url shortener feature from unyku.id

**Reply by dngda** [reply]
Just send me the rest api docs, and i'll make it for you

**Reply by dstwn** [reply]
memeng for it
